static/app/components/feedback/useFeedbackOnboarding.tsx

```diff
@@ -24,8 +24,12 @@ export function useFeedbackOnboardingSidebarPanel() {
   const organization = useOrganization();
 
   useEffect(() => {
-    if (location.hash === '#feedback-sidequest') {
+    if (
+      location.hash === '#feedback-sidequest' ||
+      location.hash === '#crashreport-sidequest'
+    ) {
       SidebarPanelStore.activatePanel(SidebarPanelKey.FEEDBACK_ONBOARDING);
+      // this tracks clicks from both feedback index and issue details feedback tab
```

Comment:
do we need to split it up in two? cc @jas-kas

Reply:
i added analytics for the separate buttons as well in this PR; this one just tracks both, so we have all 3 available to us